src/pages/Sessions.jsx

```diff
@@ -13,9 +16,25 @@ function Sessions() {
     fetchSessions();
   }, []);
 
+  const handleOpenFormDialog = () => {
+    setDisplayRegDialog(true);
+  };
+
+  const handleCloseFormDialog = () => {
+    setDisplayRegDialog(false);
+  };
+
   return (
     <>
```

Comment:
is this supposed to add spacing? Or non-intentional?

Reply:
this is a [fragment](https://reactjs.org/docs/fragments.html)! it lets us return multiple elements in a single component, without adding things to the DOM

we could've used a div too, but I don't think we need it for anything